### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krauson**

### CODE REVIEW

**Comment by yammesicka** [comment]
Please make sure you give better names for the PR, and better commit messages :)

File: app/config.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import os
```
Comment (yammesicka) [inline app/config.py:1]: Please do not upload the config.py file

File: app/go_to_date/main.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI, Request
```
Comment (yammesicka) [inline app/go_to_date/main.py:1]: Please choose an existing directory

File: app/go_to_date/static/styles.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+/* Button used to open the contact form - fixed at the bottom of the page */
+.open-button {
+  background-color: #555;
+  color: white;
+  padding: 16px 20px;
+  border: none;
+  cursor: pointer;
+  opacity: 0.8;
+  position: fixed;
+  bottom: 23px;
```
Comment (yammesicka) [inline app/go_to_date/static/styles.css:13]: Do not work with `px`, work with em or % instead :)

File: app/go_to_date/todo.py [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# 1.button
```
Comment (yammesicka) [inline app/go_to_date/todo.py:1]: Remove this file

File: app/revert_test.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+"hello world"
```
Comment (yammesicka) [inline app/revert_test.txt:1]: Remove this file, and dev.db

File: go_to_date/main3.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+from fastapi import FastAPI, Request, Form
```
Comment (yammesicka) [inline go_to_date/main3.py:1]: Please make sure you send the commit after you check all the files are relevent and ready for check.

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/go_to_date/main.py (hunk 2 above)
Comment (krauson) [reply app/go_to_date/main.py:1]: Didn't quite understand what did you mean?
what should I change?

File: README.md [hunk 7]
```diff
@@ -35,7 +35,7 @@ uvicorn app.main:app --reload
 python -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
-cp app/config.py.example app/config.py
+cp app/config.py.example app/configuration.py
```
Comment (yammesicka) [inline README.md:38]: Change back to config.py

File: app/go_to_date/main.py (hunk 2 above)
Comment (yammesicka) [reply app/go_to_date/main.py:1]: The path of the file should be app/(EXISTING_DIRECTORY_HERE)/main.py, not `go_to_date`

File: app/go_to_date/static/popup.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+function openForm() {
+    document.getElementById("myForm").style.display = "block";
```
Comment (yammesicka) [inline app/go_to_date/static/popup.js:2]: Use CSS and create class which the property `{ display: block }`, then use `toggleClass` to add/remove this class from `openForm`.

Don't edit the style directly from the JavaScript, as we try not to mix logic with styling.

File: app/go_to_date/templates/index.html [hunk 9]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="{{ url_for('static', path='/styles.css') }}" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button class="open-button" onclick="openForm()">Go to a specific date</button>
```
Comment (yammesicka) [inline app/go_to_date/templates/index.html:10]: Use JavaScript's `addEventListener`, don't use inline `onclick` event

**Review by yammesicka — CHANGES_REQUESTED** [review]

**Comment by yammesicka** [comment]
Great progress! I've added few more insights :)

File: app/go_to_date/main.py (hunk 2 above)
Comment (krauson) [reply app/go_to_date/main.py:1]: I still don't understand, should I put my feature's files directly in the "app" directory?
don't really understand what you mean by existing directory?

File: app/go_to_date/static/popup.js (hunk 8 above)
Comment (krauson) [reply app/go_to_date/static/popup.js:2]: I got a bit confused,see toggleClass belongs to jquery library which uses a different click function (not addEventListener), so should I use something like that?:
$(document).ready(function(){
    $("button").click(function(){
      $("p").toggleClass("main");

File: app/go_to_date/main.py (hunk 2 above)
Comment (yammesicka) [reply app/go_to_date/main.py:1]: There are many directories that are already exists: `routers`, `internal` etc.

Choose one of them which fits the best, according to the schema described [here](https://fastapi.tiangolo.com/tutorial/bigger-applications/), and move this file to there.

Don't create your own directory for each feature :)

File: app/go_to_date/static/popup.js (hunk 8 above)
Comment (yammesicka) [reply app/go_to_date/static/popup.js:2]: Yeah sure. I meant you should use `element.classList.toggle`

File: app/go_to_date/templates/index.html (hunk 9 above)
Comment (yammesicka) [reply app/go_to_date/templates/index.html:10]: Remove the `openForm` (and don't mark as resolved before you actually resolve it)

**Review by yammesicka — CHANGES_REQUESTED** [review]

File: app/go_to_date/main.py (hunk 2 above)
Comment (krauson) [reply app/go_to_date/main.py:1]: Ok, so if I understand it right I need to put my python file at the internal folder (and maybe change its name to go_to_date), and the content of static folder I should put in the static folder app/static, and the same goes for the templates folder?

**Pull request closed without merging**